proto/interchain_security/ccv/consumer/v1/genesis.proto

```diff
@@ -39,6 +39,12 @@ message GenesisState {
   // LastTransmissionBlockHeight nil on new chain, filled in on restart.
   interchain_security.ccv.consumer.v1.LastTransmissionBlockHeight last_transmission_block_height = 12
   [ (gogoproto.nullable) = false ];
+  // TODO: preCCV could be replaced by params.Enabled
```

Comment:
`params.Enabled` is deprecated and soon it will be removed. `preCCV` is a more suggestive name.